### PR TITLE
mobile view for questions

### PR DESCRIPTION
**Pull request by npfitz**
Epic #22826 

Mobile view for Questions. Please note that app bar adjustments are included in https://github.com/metabase/metabase/pull/23814

Before:
image
After: 
image

### CODE REVIEW

**Comment by kulyk** [comment]
It'd be cool to extend the info sidebar's width so it takes the whole horizontal space when open:

<details>
<summary>Screenshot</summary>
CleanShot 2022-07-11 at 14 36 47@2x
</details>

**Review by kulyk — COMMENTED** [review]
Nice work, loving it's solved primarily via CSS 👍 Having a question about the notebook editor and some feedback re: the info sidebar I'd like to sort out

File: frontend/src/metabase/query_builder/components/view/ViewHeader.jsx [hunk 1]
```diff
@@ -484,8 +481,8 @@ function ViewTitleHeaderRightSide(props) {
       {hasExploreResultsLink && <ExploreResultsLink question={question} />}
       {hasRunButton && (
         <RunButtonWithTooltip
-          className={cx("text-brand-hover text-dark hide", {
-            "sm-show": !isShowingNotebook || isNative,
+          className={cx("text-brand-hover text-dark", {
+            "sm-show": isNative,
             "text-white-hover": isResultDirty,
           })}
```
[the remark this answers is not in the document]
Comment (npfitz) [reply frontend/src/metabase/query_builder/components/view/ViewHeader.jsx:487]: Whoops. Yeah, I should put that back. I was trying to figure out why the run button wasn't showing up in the mobile view.

**Review by kulyk — APPROVED** [review]
LGTM! Please don't forget to fix the "run" button for notebook mode

**Pull request merged**